REFERENCE.md

```diff
@@ -18,7 +18,7 @@
 * [`delete_kotsadm`](#delete_kotsadm): Delete the Kots admin-console applicaiton from the cluster. Note, if you use this on a Kurl host, you will need to re-run the Kurl installer 
 * [`destroy_nginx_ingress`](#destroy_nginx_ingress): Tear down the Nginx IngressController.
 * [`get_kots_app_status`](#get_kots_app_status): Return the state of a given Kots application, or not-installed. Will also return not-installed if kots itself is not installed.
-* [`kots_download`](#kots_download): Downloads the currently installed source of a given Kots application from the admin console to the given directory.
+* [`kots_download`](#kots_download): Downloads the currently installed source of a given Kots application from the admin console to the given directory. This task is a wrapper ro
```

Comment:
Incomplete sentence.

Reply:
Yeah, that seems to be a puppet-strings bug in handling the short description of the entry lists. You can see it in the REFERENCE file. The full description is present under the full function/task/plan entry further down.